game/src/edit/traffic_signals/mod.rs

```diff
@@ -795,21 +795,21 @@ fn check_for_missing_turns(app: &App, members: &BTreeSet<IntersectionID>) -> Opt
     }
 
     let mut bundle = BundleEdits::get_current(app, members);
-    // Stick all the missing turns in a new phase at the beginning.
+    // Stick all the missing turns in a new stage at the beginning.
     for signal in &mut bundle.signals {
-        let mut phase = Phase::new();
+        let mut stage = Stage::new();
         // TODO Could do this more efficiently
         for g in &all_missing {
             if g.parent != signal.id {
                 continue;
             }
             if g.crosswalk {
-                phase.protected_groups.insert(*g);
+                stage.protected_groups.insert(*g);
```

Comment:
Based on my new understanding, what we're currently calling "group", as in "turn group" and "protected_group", more closely aligns with the standard usage of "Phase". Is that right?

Reply:
I think so, but I'm confused by https://en.wikipedia.org/wiki/Traffic_light_control_and_coordination#Phases_and_stages. There, a phase might be multiple turn groups with a single origin. If that's the case, we should keep the "turn group" deliberately different.

Reply:
From [our beloved manual](https://ops.fhwa.dot.gov/publications/fhwahop08024/chapter4.htm),
> ...a phase is defined as a controller timing unit associated with the control of one or more movements...

I don't like the name "turn group", but I think you're right that it's not a phase, because we need to leave open the possibility that a phase has multiple "turn groups".

How do we feel about renaming "turn group" to "movement"?

The signal control code I'm writing has one timer per turn group, but that's only because we don't have distinct `phase` object which associates multiple turn groups.

Reply:
| old  | new |  future |
|------------|--------------|------|
| `turn_group` | `turn_group` | `turn_group`/`movement`/`?` |
| (doesn't exist) | (doesn't exist) | `phase` |
| `phase` | `stage` | `stage` |

Reply:
+1 to this plan. I'll send another PR for `TurnGroup` -> `Movement`. Establishing some kind of `Phase` as a `BTreeSet<MovementID>` likely makes sense after that.

 To check, are y'all OK with an abbreviation of `mvmnt` for variable names, similar to how `tg` is used now? Or any other stylistic conventions suggested?

Reply:
I've spent the last several years writing Objective-C, I like long names like: "movement", "intersection_movement", and "variable_that_represents_movement_for_the_intersection" 🙃